nbdime/webapp/nbdimeserver.py

```diff
@@ -285,19 +309,23 @@ def make_app(**params):
     else:
         prefix = ''
 
+    env = Environment(loader=FileSystemLoader([template_path]), autoescape=False)
     settings = {
-        "static_path": static_path,
-        "static_url_prefix": prefix + '/static/',
-        "template_path": template_path,
+        'static_path': static_path,
+        'static_url_prefix': prefix + '/static/',
+        'template_path': [template_path],
+        'base_url': base_url,
+        'jinja2_env': env,
+        'mathjax_url': 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js',
```

Comment:
since notebook is added as a dependency, we cna get the mathjax from there:

- notebook.DEFAULT_STATIC_FILES_PATH is the static directory
- mathjax is in `components/mathjax/MathJax.js` relative to there

Could be a StaticFileHandler serving `/static/mathjax` from `DEFAULT_STATIC_FILES_PATH/components/mathjax`, with mathjax_url = `static_url('mathjax/MathJax.js')`

Reply:
Added.